graylog2-server/src/main/java/org/graylog/schema/EventFields.java

```diff
@@ -24,6 +24,7 @@
     public static final String EVENT_ERROR_DESCRIPTION = "event_error_description";
     public static final String EVENT_LOG_NAME = "event_log_name";
```

Comment:
Do not see one for `event_error_code` in here.  On the Global Protect Page

Reply:
It's up on line 23.  Not showing up in the review because it wasn't changed.